LaserGRBL/GrblFile.cs

```diff
@@ -145,10 +165,7 @@ public override string ToGCodeNumber(ref int cumX, ref int cumY, L2LConf c)
 			{
 				cumX += mPixLen;
 
-				if (c.pwm)
-					return string.Format("X{0} S{1}", formatnumber(cumX, c.oX, c), mColor);
-				else
-					return string.Format("X{0} {1}", formatnumber(cumX, c.oX, c), Fast(c) ? c.lOff : c.lOn);
+				return string.Format("X{0} {1}", formatnumber(cumX, c.oX, c), formatlaserpower(mColor,c));
```

Comment:
I prefer the explicit if/else on the external ToGCodeNumber function that apply two different format string if PWM or not, instead of a single "formatlaserpower" that does both internally